beets/autotag/hooks.py

```diff
@@ -609,25 +619,17 @@ def album_candidates(items, artist, album, va_likely, extra_tags):
     constrain the search.
     """
 
+    common_args = [album, len(items), extra_tags]
     # Base candidates if we have album and artist to match.
     if artist and album:
-        try:
-            yield from mb.match_album(artist, album, len(items),
-                                      extra_tags)
-        except mb.MusicBrainzAPIError as exc:
-            exc.log(log)
+        yield from handle_exc(mb.match_album, artist, *common_args)
```

Comment:
In my opinion, this pattern would be much more readable as

```python
    if artist and album and config["musicbrainz"]["enabled"]:
        yield from handle_exc(mb.match_album, artist, *common_args)
```

with the check for MB being enabled removed from `handle_exc`. This would separate exception handling and checking whether to query musicbrainz somewhat more cleanly.

If not going this way, renaming `handle_exc` -> `invoke_mb` or similar would also help.

Reply:
Good point, @wisp3rwind. I've just renamed the function to `invoke_mb`